Decred Wallet/Features/Receive/ReceiveViewController.swift

```diff
@@ -65,11 +62,48 @@ class ReceiveViewController: UIViewController,UIDocumentInteractionControllerDel
         super.viewWillDisappear(animated)
         self.navigationItem.leftBarButtonItem = nil
     }
+    
+    @objc func accountSelectionDropdownBtnClick() {
+        self.accountSelectionDropdownBtn.isSelected = !self.accountSelectionDropdownBtn.isSelected
+        
+        if self.receiveAccountTableView == nil {
+            self.createReceiveAccountTableView()
+        }
+        
+        self.receiveAccountTableView.showView()
+    }
+    
+    private func createReceiveAccountTableView() {
+        self.receiveAccountTableView = AccountTableView.init(frame: CGRect(x: 0, y: 0, width: self.view.frame.width, height: self.view.frame.height))
+        UIApplication.shared.keyWindow?.addSubview(self.receiveAccountTableView!)
+        
+        self.receiveAccountTableView.selectedAccount = {
+            [weak self] (account:DcrlibwalletAccount,walletName:String) in
+            
+            guard let this = self else { return }
+            
+            this.myacc = account
+            
+            this.selectedAccountNameLbl.text = this.myacc?.name
+            this.selectedAccountWalletNameLbl.text = walletName
+            
+            let total = "\(this.myacc?.balance?.total ?? 0)"
+            this.selectedAccountTotalBalanceLbl.attributedText = Utils.getAttributedString(str: "\(total)", siz: 14.0, TexthexColor: UIColor.init(hex: "0A1440"))
+                            
+            this.getAddress(accountNumber: (this.myacc!.number))
+        }
+        self.receiveAccountTableView.hide = {
+            [weak self] () in
+            guard let this = self else { return }
+            this.accountSelectionDropdownBtn.isSelected = !this.accountSelectionDropdownBtn.isSelected
+        }
+    }
 
     func setupExtraUI() {
-        self.imgWalletAddrQRCode.addGestureRecognizer(tapToCopyAddressGesture())
-        self.lblWalletAddress.addGestureRecognizer(tapToCopyAddressGesture())
-        self.accountDropdown.backgroundColor = UIColor.white
+        self.accountSelectionDropdownBtn.addTarget(self, action: #selector(accountSelectionDropdownBtnClick), for: .touchUpInside)
+        self.selectedAccountTotalBalanceLbl.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(accountSelectionDropdownBtnClick)))
```

Comment:
```suggestion

self.selectedAccountTotalBalanceLbl.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(accountSelectionDropdownBtnClick)))
```

Plese delete whitespace here and other places too